src/manager.h

```diff
@@ -66,7 +66,7 @@ class Manager
    *   @return Return true if bus is not faulted, otherwise a bus fault is
    * active.
    */
-  bool Process();
+  bool Process(double external_time=0.0);
```

Comment:
document the external_time param addition with `@param`

Reply:
Added